keyboards/h87a/keymaps/gam3cat/config.h

```diff
@@ -0,0 +1,9 @@
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary. 

And you should have `#pragma once` at the top, as an include guard. 

```suggestion
#pragma once
```